docs/User-Experience/Research-Syntheses/2024, Summer - OFA Admin Experience.md

```diff
@@ -0,0 +1,62 @@
pages default to filtering results to "Newest". This ticket updates the language of that filter to "Most recent version" and adds that behavior to the Data Files page.</td><td>Search Indexes, Data Files</td><td><a href="https://github.com/raft-tech/TANF-app/issues/3087">#3087</a></td><td>4.0 / P4</td></tr><tr><td>Add multiselect control to Search Indexes Fiscal Period Filter</td><td>Currently the filter control on DAC Search Indexes pages is a single option dropdown. This ticket replaces it with the multiselect control that we use when filtering by STT. This replicates the SQL union queries used in the legacy system.</td><td>Search Indexes</td><td><a href="https://github.com/raft-tech/TANF-app/issues/3102">#3102</a></td><td>4.0 / P4</td></tr><tr><td>Add filters from DAC Data Files page to Data File Summaries pages</td><td>Currently Data File Summaries pages lack filtering capability. This ticket delivers filter options matching those on the Data Files page.</td><td>Data File Summaries</td><td><a href="https://github.com/raft-tech/TANF-app/issues/3093">#3093</a></td><td>4.0 / P2</td></tr><tr><td>[Spike] Investigate adding Change Message types</td><td>This spike investigates whether we can supplement the current Change Message column on log entries with a change type to allow for filtering capability. </td><td>Log Entries</td><td><a href="https://github.com/raft-tech/TANF-app/issues/3092">#3092</a></td><td>4.0 /<a data-footnote-ref href="#user-content-fn-8"> P</a>2</td></tr><tr><td>[Spike] Investigates how we can provide a tabular view of data file summaries</td><td>Currently data file summaries are served up in a raw text format. To make them easier to read we should investigate how we might map these data to a table view.</td><td>Data File Summaries (Specific Summary view)</td><td><a href="https://github.com/raft-tech/TANF-app/issues/3095">#3095</a></td><td>4.0 / P2</td></tr><tr><td>Rearrange Data Files filters and implement multiselect control for STT filtering</td><td>Adds the multiselect control for STT filtering on the Data Files page and rearanges filters into a more intuitive order.</td><td>Data Files</td><td><a href="https://github.com/raft-tech/TANF-app/issues/3097">#3097</a></td><td>4.0 / P2</td></tr><tr><td>[Spike] Investigate YYYYMMDD value filtering for data files</td><td>Higher lift ideal solution following on from #3076. Investigates how we might add the ability to filter the DAC Data Files page to those submitted on a specific date or within a specific date range. </td><td>Data Files</td><td><a href="https://github.com/raft-tech/TANF-app/issues/3077">#3077</a></td><td>Beyond 4.0 / P2</td></tr><tr><td>List of Cat 1,4 rejected case numbers</td><td>Provides a method of filtering current django outputs to produce a list of case numbers on rejected data files.</td><td>Data File Summaries (Specific Summary view)</td><td><a href="https://github.com/raft-tech/TANF-app/issues/3096">#3096</a></td><td>Beyond 4.0 / P2</td></tr></tbody></table>
```

Comment:
re: #3095, my impression is that the summaries are in json format.

Reply:
re: #3097 -- confirmed with Yun that she does not need multiselect on STTs on data files page and also wants to combine year and quarter into one fiscal period filter (as shown in search indexes pages.

Reply:
re: #3096 -- the explanation needs clarity. OFA groups want  to be able to see which case/months are associated with cat 1 and cat 4 errors. please see legacy system transmission emails for reference.